### ISSUE TITLE
sirepo with GPUs

### ISSUE THREAD

**Issue by moellep** [radiasoft/sirepo#4560]
Using a GPU to run ml image processing with sirepo/activait would be ideal. Could sirepo be allocated a GPU which could be shared across users as a resource?

**Comment by robnagler** [comment]
Yes, we could allocate GPUs with docker. This would be dynamic, and it might have to be a different job/agent type, because the GPU would have to be released at the end of the job. The pools might have to be different, too.
